src/main/java/com/bunq/sdk/model/generated/object/DraftPaymentResponse.java

```diff
@@ -0,0 +1,47 @@
+package com.bunq.sdk.model.generated.object;
+
+import com.bunq.sdk.model.BunqModel;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ */
```

Comment:
Intended that this class has an empty comment doc ?

Reply:
generated